security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -155,23 +159,36 @@ protected TableFormatterConfig createTableFormatterConfig() {
         return line.hasOption(option) ? Optional.of(line.getOptionValue(option)) : Optional.empty();
     }
 
+    private static void copyBytesToPath(byte[] bytes, Path dest) {
+        try {
+            Files.write(dest, bytes);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
```

Comment:
Does this method should be moved to an utility class in commons?

Reply:
We could move this into commos when we use it in another case.